src/ClientCommand.cpp

```diff
@@ -758,29 +759,38 @@ void CClient::UserCommand(CString& sLine) {
             pMod->MoveRegistry(sNewModPath);
         }
 
-        CString sNetworkAddError;
-        CIRCNetwork* pNewNetwork =
-            pNewUser->AddNetwork(sNewNetwork, sNetworkAddError);
+        if (!sOldUser.Equals(sNewUser) ||
+            !pOldNetwork->GetName().Equals(sNewNetwork)) {
+            CString sNetworkAddError;
+            CIRCNetwork* pNewNetwork =
+                pNewUser->AddNetwork(sNewNetwork, sNetworkAddError);
 
-        if (!pNewNetwork) {
-            PutStatus(t_f("Error adding network: {1}")(sNetworkAddError));
-            return;
-        }
+            if (!pNewNetwork) {
+                PutStatus(t_f("Error adding network: {1}")(sNetworkAddError));
+                return;
+            }
 
-        pNewNetwork->Clone(*pOldNetwork, false);
+            pNewNetwork->Clone(*pOldNetwork, false);
+        } else {
+            CIRCNetwork* pOldNetwork = pNewUser->FindNetwork(sOldNetwork);
```

Comment:
Same comment as in webadmin: this would be more understandable with the reverse condition and a comment